EXAMPLES/PYARPACK/pyarpackSparseLU.py.in

```diff
@@ -113,4 +122,4 @@ for val, vec in zip(arpackSlv.val, arpackSlv.vec):
     print("eigen value:", val)
     print("eigen vector:")
     for v in range(n):
-        print(vec[v])
+        print(vec[v])
```

Comment:
I guess this is no op! Can you remove it and update the changelog? Thanks!

Reply:
Did you mean restoring the EOL ? If so I restored it in the newer commit. Also fixed the solver parameters to match their respective solver (slvOffset and slvScale moved back to SparseLDLT, slvPvtThd to SParseLU, etc)

Reply:
thanks